src/python-common/ceph/deployment/drive_group.py

```diff
@@ -123,29 +190,70 @@ def __init__(self,
         self.osd_id_claims = osd_id_claims
 
     @classmethod
-    def from_json(self, json_drive_group):
+    def from_json(cls, json_drive_group):
```

Comment:
I would include the filters and size validation in the validation method

Reply:
agree